### PR TITLE
Add replace for moby v20.10.0-beta1 to fix build issue on darwin

### PR DESCRIPTION
**Pull request by rossf7**
Towards giantswarm/giantswarm#13905

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md
Comment (rossf7) [inline CHANGELOG.md:12]: @yasn77 Thanks again for adding this. I'm going to cut a 3.1.0 release once this is merged.

I just removed the link as normally we don't include since the issue is private.

File: go.mod
Comment (rossf7) [inline go.mod:33]: @tomahawk28 I've moved this to a replace. As otherwise I worry we might miss reverting to the buggy version.

WDYT?

Comment (tomahawk28) [reply go.mod:33]: Good, Keep both in `require` and `replace` would work.

**Review by tomahawk28 — APPROVED** [review]

File: go.mod
Comment (rossf7) [reply go.mod:33]: I just bumped the orb to the latest too. To get that out of the way. I'll merge once CI is green.

Comment (rossf7) [reply go.mod:33]: Nancy finds a ton of stuff :( 

https://app.circleci.com/pipelines/github/giantswarm/helmclient/366/workflows/4ef17a1d-9972-4fe3-9a27-558e7ad40f50/jobs/3600

I've reverted and will fix it separately.

**Pull request merged**
